tests/apollo/test_skvbc_dbsnapshot.py

```diff
@@ -258,6 +254,8 @@ async def test_create_dbcheckpoint_cmd(self, bft_network, tracker):
             assert(len(res[0].response.db_checkpoint_info) == 1)
             dbcheckpoint_info_list = res[0].response.db_checkpoint_info
             assert(any(dbcheckpoint_info.seq_num == 300 for dbcheckpoint_info in dbcheckpoint_info_list))
+            # Verify that last dbCheckpoint is created with last block id 200.
+            assert(any(dbcheckpoint_info.block_id == 200 for dbcheckpoint_info in dbcheckpoint_info_list))
```

Comment:
What i meant is that you can call verify_snapshot_is_available(snapshot_id) to verify if checkpoint dir is created in the file system.

Reply:
fixed.